src/python/pants/option/arg_splitter.py

```diff
@@ -213,12 +214,22 @@ def assign_flag_to_scope(flg: str, default_scope: str) -> None:
         if not goals and not self._help_request:
             self._help_request = NoGoalHelp()
 
+        # When we remove this deprecation, this case will turn into an error.
+        deprecated_conditional(
+            lambda: bool(passthru) and len(passthru_owners) > 1,
+            removal_version="1.31.0.dev0",
+            entity_description="Ambiguous scopes for passthrough args",
+            hint_message="In future, all scopes which accept passthrough args (arguments after --) "
+            f"will receive them, and passing multiple scopes (in this case: {passthru_owners}) "
+            "while using passthrough args will no longer be allowed.",
```

Comment:
> In the future, you will no longer be able to use passthrough arguments when there are multiple owners because it results in ambiguity. (In this case, these owners found: {}).